main.go

```diff
@@ -25,6 +25,7 @@ func init() {
 func main() {
 	http.HandleFunc("/resources.json", resourcesHandler)
 	http.Handle("/", http.StripPrefix("/", http.FileServer(http.Dir("./static"))))
+	log.Print("Listening on port 80...")
```

Comment:
til `Print` ~= `Println`, in that both add `\n` ... https://groups.google.com/forum/#!topic/golang-nuts/SOibvyF6Tjo